src/main/java/seedu/task/logic/commands/AddCommand.java

```diff
@@ -0,0 +1,25 @@
+package seedu.task.logic.commands;
+
+/**
+ * Abstract class to represent generic add operations.  
+ * @author kian ming
+ */
+
+public abstract class AddCommand extends Command {
+    
+    public static final String COMMAND_WORD = "add";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task or event to the task book.\n"
+            + "Parameters: TASK_NAME /desc DESCRIPTION /by DEADLINE\n"
+            + "Example: " + COMMAND_WORD
+            + " CS2103 Lab 6 /desc hand in through codecrunch /by 30-12-16\n"
+            + "Parameters: EVENT_NAME /desc DESCRIPTION /from DURATION\n"
+            + "Example: " + COMMAND_WORD
+            + " CS2103 Lab 6 /desc hand in through codecrunch /from 30-12-16 > 31-12-16\n"
```

Comment:
Please note on the format of our duration, it does not have to be DD-MM-YY anymore.